### DISCUSSION TITLE
How to set video options?

### DISCUSSION THREAD

**Discussion by laugepetersen** [fancyapps/ui#79] [Q&A]
Hi there,

I can't seem to make it work for the options on the media type.

Example:

I tried change format on videos in general as your docs and added the tester class "lauge" to html5 video elements. Nothing seems to work except the "parentEl" option.

![image](https://user-images.githubusercontent.com/24874516/130191398-04838f40-b59d-484d-a822-af14645afebb.png)

It also happens, that the other options such as these below doesn't console.log anything and is directly copied from the docs.
![image](https://user-images.githubusercontent.com/24874516/130192581-126e8f1b-ff57-4100-a909-7586cfbfcb7f.png)

**Comment by fancyapps** [comment]
Simply nest your options for Html plugin inside `Html` option, like this:

```JS
Fancybox.bind("#manual", {
  Html : {
    html5video : {
      tpl: `<video class="MY_CUSTOM_CLASS fancybox__html5video" playsinline controls controlsList="nodownload" poster="{{poster}}">
      <source src="{{src}}" type="{{format}}" />
      Sorry, your browser doesn\'t support embedded videos, <a href="{{src}}">download</a> and watch with your favorite video player!
    </video>`,
      format: "",
    }
  }
});
```
